device/thunder_ripple_sdk/src/processors/thunder_device_info.rs

```diff
@@ -664,11 +656,32 @@ impl ThunderDeviceInfoRequestProcessor {
     async fn get_all_timezones(
         state: &ThunderState,
     ) -> Result<ThunderAllTimezonesResponse, RippleError> {
+        let params = Some(DeviceChannelParams::Json(
+            json!({
+                "timeZones": [
+                    "America/New_York",
```

Comment:
We need to use the current timezone value we get from the timezone DST as the input for this call and not hardcode this